### PR TITLE
Proper handling of unofficial plugins

### PR DESCRIPTION
**Pull request by karensawrey**
WIP

Commit #1:
Adding missing commas, removing extra commas, fixing typos, removing "we" from the tutorial to comply with the style guide.

### CODE REVIEW

File: pages/plugins.md.erb [hunk 1]
```diff
@@ -8,16 +8,16 @@ Add plugins to [command steps](/docs/pipelines/command-step) in your YAML pipeli
 
 ## What is a plugin?
 
-A plugin is a Git repository that overrides or extends a [command step](/docs/pipelines/command-step)’s behavior using one or more [hooks](/docs/agent/v3/hooks). To use a plugin, you reference it in your pipeline YAML configuration, and when that step is run your agent will clone and execute each of the plugin’s [hooks](/docs/agent/v3/hooks).
+A plugin is a Git repository that overrides or extends a [command step](/docs/pipelines/command-step)’s behavior using one or more [hooks](/docs/agent/v3/hooks). To use a plugin, you reference it in your pipeline YAML configuration, and when that step is run, your agent will clone and execute each of the plugin’s [hooks](/docs/agent/v3/hooks).
```
Comment (plaindocs) [inline pages/plugins.md.erb:11]:
```suggestion
Plugins are small self-contained pieces of extra functionality that help you customize Buildkite to your specific workflow. Plugins modify your build [command steps](/docs/pipelines/command-step) at one or more of the ten [job lifecycle hooks](/docs/agent/v3/hooks). Each hook changes a different part of how your jobs set up the environment, check out code, run commands, handle artifacts or clean up the environment. 

Reference plugins in your pipeline configuration, and when the step containing the plugin runs, your agent will override the default behavior of with the hooks defined in the plugin [hooks](/docs/agent/v3/hooks).
```

Sneaky fix for DOC-190 as well

**Review by plaindocs — APPROVED** [review]
These are all good changes.

- [ ] Take a quick look at my suggestion, and if you're happy with it, merge it in and :shipit:

File: pages/plugins.md.erb (hunk 1 above)
Comment (karensawrey) [reply pages/plugins.md.erb:11]: Ha, nice sneaky rewrite that tackles another issue :) Love it!

File: pages/plugins.md.erb [hunk 2]
```diff
@@ -8,16 +8,18 @@ Add plugins to [command steps](/docs/pipelines/command-step) in your YAML pipeli
 
 ## What is a plugin?
 
-A plugin is a Git repository that overrides or extends a [command step](/docs/pipelines/command-step)’s behavior using one or more [hooks](/docs/agent/v3/hooks). To use a plugin, you reference it in your pipeline YAML configuration, and when that step is run your agent will clone and execute each of the plugin’s [hooks](/docs/agent/v3/hooks).
+Plugins are small self-contained pieces of extra functionality that help you customize Buildkite to your specific workflow. Plugins modify your build [command steps](/docs/pipelines/command-step) at one or more of the ten [job lifecycle hooks](/docs/agent/v3/hooks). Each hook changes a different part of how your jobs set up the environment, check out code, run commands, handle artifacts or clean up the environment.
```
Comment (plaindocs) [inline pages/plugins.md.erb:11]:
```suggestion
Plugins are small self-contained pieces of extra functionality that help you customize Buildkite to your specific workflow. Plugins modify your build [command steps](/docs/pipelines/command-step) at one or more of the ten [job lifecycle hooks](/docs/agent/v3/hooks). Each hook changes a different part of how your jobs set up the environment, check out code, run commands, handle artifacts or clean up the environment.
```

**Pull request merged**